conans/model/info.py

```diff
@@ -575,6 +575,27 @@ def header_only(self):
         self.options.clear()
         self.requires.clear()
 
+    def msvc_compatible(self):
+        if self.settings.compiler != "msvc":
+            return
+
+        compatible = self.clone()
+        version = compatible.settings.compiler.version
+        runtime = compatible.settings.compiler.runtime
+        runtime_type = compatible.settings.compiler.runtime_type
+
+        compatible.settings.compiler = "Visual Studio"
+        version = str(version)[:4]
+        _visuals = {'19.0': '14',
+                    '19.1': '15',
+                    '19.2': '16'}
+        compatible.settings.compiler.version = _visuals[version]
```

Comment:
may raise, need a sanity check

Reply:
Sure, this is a preliminary implementation, this needs to be improved.